File: subdir/hello.c
// 2026-05-24
#include <stdio.h>
#include <time.h>

int main(void)
{
    time_t now;
    struct tm *tm_now;
    char date[11];

    printf("Hello world\n");
    now = time(NULL);
    tm_now = localtime(&now);
    if (tm_now != NULL && strftime(date, sizeof(date), "%Y-%m-%d", tm_now) > 0) {
        printf("%s\n", date);
    } else {
        printf("none\n");
    }
    return 0;
}

File: subdir/hello.f90
! 2026-05-24
program hello
    character(len=10) :: date
    integer :: values(8), stat

    print *, "Hello world"
    call date_and_time(date=date, values=values)
    stat = values(1)
    if (stat > 0) then
        print *, date(1:4) // "-" // date(5:6) // "-" // date(7:8)
    else
        print *, "none"
    end if
end program hello
